guides/v2.1/install-gde/prereq/dev_install.md

```diff
@@ -15,39 +15,58 @@ Totally lost? Need a helping hand? Try our [installation quick reference (tutori
 
 ## Intended audience {#integrator-aud}
 
-The audience for this topic is anyone who contributes to the {{site.data.var.ce}} codebase. You should be highly technical, understand {% glossarytooltip d85e2d0a-221f-4d03-aa43-0cda9f50809e %}Composer{% endglossarytooltip %} and Git commands, and be able to upgrade the Magento system software and extensions using those commands. If that isn't you, go back and [choose another starting point]({{ page.baseurl }}/install-gde/bk-install-guide.html).
+The audience for this topic is anyone who contributes to the {{site.data.var.ce}} codebase.
+You should be highly technical, understand {% glossarytooltip d85e2d0a-221f-4d03-aa43-0cda9f50809e %}Composer{% endglossarytooltip %} and Git commands, and be able to upgrade the Magento system software and extensions using those commands. If that isn't you, go back and [choose another starting point]({{ page.baseurl }}/install-gde/bk-install-guide.html).
 
 {:.bs-callout .bs-callout-warning}
-If you clone the Magento 2 GitHub repository, you _cannot_ use the Magento software in a production environment. You cannot have a live store that accepts orders and so on.
+If you clone the Magento 2 GitHub repository, you _cannot_ use the Magento software in a production environment.
+You cannot have a live store that accepts orders and so on.
 
 ## Prerequisites
+
 {% include install/prereq.md %}
 
 {% include install/composer-overview.md %}
 
 ## Install Composer {#instgde-prereq-compose-install}
+
 {% include install/composer-clone.md %}
 
 ## Clone the Magento repository {#instgde-prereq-compose-clone}
 
-This section discusses how to get current code by cloning [Magento's GitHub repository](https://github.com/magento/magento2){:target="_blank"} and checking out branches. You can either checkout a release branch or a development branch:
+This section discusses how to get current code by cloning [Magento's GitHub repository][]{:target="_blank"} and checking out branches.
+You can either checkout a release branch or a development branch:
 
-*	Release branches, like `2.1.0`, are more stable. You _must_ use a release branch with the [Data Migration Tool]({{page.baseurl}}/migration/bk-migration-guide.html).
+* Release branches, like `2.1.0`, are more stable. You _must_ use a release branch with the [Data Migration Tool][].
 
-*	Development branches, like `2.2-develop`, contain the latest changes.
+* Development branches, like `2.2-develop`, contain the latest changes.
 
 Currently, the `2.2-develop` branch is the default, but you can checkout a release branch, like `2.1.0`, after cloning.
```

Comment:
Currently, the 2.3 branch is default. It makes sense to eliminate the name of current branch, because it can change each major release.